package.json

```diff
@@ -400,9 +400,100 @@
 							"verticalOffset": 0
 						}
 					}
+				},
+				"cursorless.wrapperSnippetPreferences": {
+					"description": "Allows snippets to signal what their default scope types etcetera should be",
+					"type": "object",
+					"scope": "language-overridable",
+					"default": {
+						"cursorless.wrapper.ifStatement": {
+							"scopeType": "statement"
+						},
+						"cursorless.wrapper.tryCatchStatement": {
+							"scopeType": "statement"
+						},
+						"cursorless.wrapper.ifElseStatementIfBranch": {
+							"scopeType": "statement"
+						},
+						"cursorless.wrapper.ifElseStatementElseBranch": {
+							"scopeType": "statement"
+						}
+					},
+					"additionalProperties": {
+						"type": "object",
+						"properties": {
+							"scopeType": {
+								"type": "string",
+								"enum": [
```

Comment:
This enum will get stale, but I don't think it will cause any serious problems, and the autocomplete will be nice